### ISSUE TITLE
About the performance on VLCS dataset

### ISSUE THREAD

**Issue by JWarlock** [fmcarlucci/JigenDG#10]
I've found that the default parameters setting produces result about 2 points higher on VLCS dataset, than the result reported in your original paper. Did you just find a better parameters setting, or was it because the tricky train-test split of VLCS dataset in the code?

**Comment by fmcarlucci** [comment]
Hi, how many times did you run the experiment? We run each experiments multiple times and report the average, that's why you might have found a higher score on a specific run.

**Comment by dailenson** [comment]
hi, I want to ask what is your script to train the VLCS dataset? I don't know the parameters settings.

**Comment by yunkaiyu** [comment]
Hi, may i confirm the train-test split protocol? The VLCS dataset is divided to training set (70%) and a test set (30%) by random selection as reported in your work. However, training set: validation set(crossval): test set $\approx$ 9:1:9  from your VLCS copy. So do we need to re-divide the VLCS (full)? Thanks!

**Comment by fmcarlucci** [comment]
Hi, sorry for the late replies - I no longer have access to all my files, so I'll have to go by memory.

@yunkaiyu I would trust the splits defined inside the JigenDG/data/txt_lists/ for the VLCS splits

@dailenson I think the specific parameters should be: 30 epochs, batch size 128, l r0.001 jigsaw classes 30 jigsaw weight0.5, bias0.8
